editor/gui/scene_tree_editor.cpp

```diff
@@ -925,6 +925,24 @@ void SceneTreeEditor::_notification(int p_what) {
 
 			_update_tree();
 		} break;
+
+		case NOTIFICATION_VISIBILITY_CHANGED: {
+			if (is_visible()) {
+				TreeItem *item = nullptr;
+				if (selected) {
+					// Scroll to selected node.
+					item = _find(tree->get_root(), selected->get_path());
+				} else if (marked.size() == 1) {
+					// Scroll to a single marked node.
+					Node *marked_node = *marked.begin();
+					item = marked_node ? _find(tree->get_root(), marked_node->get_path()) : nullptr;
```

Comment:
`item` is intialized as `nullptr` so this seems superfluous.
```suggestion
					if (marked_node) {
						item = _find(tree->get_root(), marked_node->get_path());
					}			
```